### DISCUSSION TITLE
checkbox点击一个全部选中

### DISCUSSION THREAD

**Discussion by Poom6zZ** [element-plus/element-plus#16783] [Q&A]
自己的工程里点击一个全部选中
![image](https://github.com/element-plus/element-plus/assets/56908383/ae858825-7160-4013-bd98-0a0f05fd26d3)
在playground里却是可以单独点选的
https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6Ijx0ZW1wbGF0ZT5cbiAgPGVsLWNoZWNrYm94LWdyb3VwIHYtbW9kZWw9XCJjaGVja2JveEdyb3VwXCI+XG4gICA8ZWwtY2hlY2tib3ggdi1mb3I9XCJpdGVtIGluIGRpYWxvZ1VzZXJEYXRhXCIgOmtleT1cIml0ZW0uaWRcIiA6dmFsdWU9XCJpdGVtLmlkXCI+XG4gICAgICAgICAgICAgICAge3sgaXRlbS51c2VybmFtZSB9fe+8iHt7IGl0ZW0ucm9sZUNuTmFtZSB9fe+8iVxuICAgICAgICAgICAgICA8L2VsLWNoZWNrYm94PiAgICAgICAgICAgXG4gIDwvZWwtY2hlY2tib3gtZ3JvdXA+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmIH0gZnJvbSAndnVlJ1xuXG5jb25zdCBjaGVja2JveEdyb3VwID0gcmVmKFsnMSddKVxubGV0IGRpYWxvZ1VzZXJEYXRhID0gW1xuICAgICAgICB7XG4gICAgICAgICAgICBcInJvbGVDbk5hbWVcIjogXCLnrqHnkIblkZhcIixcbiAgICAgICAgICAgIFwicGFzc3dvcmRcIjogXCJcIixcbiAgICAgICAgICAgIFwiY29kZVwiOiBcIjFcIixcbiAgICAgICAgICAgIFwiY3JlYXRlVGltZVwiOiBcIjIwMjMtMTAtMjdUMTY6MDE6MDBcIixcbiAgICAgICAgICAgIFwiY3JlYXRlVXNlclwiOiBcImFkbWluXCIsXG4gICAgICAgICAgICBcImlkXCI6IDEsXG4gICAgICAgICAgICBcImlzRGVsXCI6IDAsXG4gICAgICAgICAgICBcInVzZXJuYW1lXCI6IFwiYWRtaW5cIixcbiAgICAgICAgICAgIFwic3RhdHVzXCI6IDFcbiAgICAgICAgfSxcbiAgICAgICAge1xuICAgICAgICAgICAgXCJyb2xlQ25OYW1lXCI6IFwi5rWL6K+V5ZGYXCIsXG4gICAgICAgICAgICBcInBhc3N3b3JkXCI6IFwiXCIsXG4gICAgICAgICAgICBcImNvZGVcIjogXCIyXCIsXG4gICAgICAgICAgICBcImNuTmFtZVwiOiBcIuW6nlwiLFxuICAgICAgICAgICAgXCJjcmVhdGVUaW1lXCI6IFwiMjAyMy0xMC0yN1QxNjowMTowMFwiLFxuICAgICAgICAgICAgXCJ0ZWxcIjogMTIzLFxuICAgICAgICAgICAgXCJjcmVhdGVVc2VyXCI6IFwiYWRtaW5cIixcbiAgICAgICAgICAgIFwiaWRcIjogMixcbiAgICAgICAgICAgIFwiaXNEZWxcIjogMCxcbiAgICAgICAgICAgIFwiZW1haWxcIjogXCIyMTIxM0BxcS5jb21cIixcbiAgICAgICAgICAgIFwidXNlcm5hbWVcIjogXCJwb29tXCIsXG4gICAgICAgICAgICBcInN0YXR1c1wiOiAxXG4gICAgICAgIH0sXG4gICAgICAgIHtcbiAgICAgICAgICAgIFwicm9sZUNuTmFtZVwiOiBcIuOAiuacquWIhumFjeOAi1wiLFxuICAgICAgICAgICAgXCJwYXNzd29yZFwiOiBcIlwiLFxuICAgICAgICAgICAgXCJjb2RlXCI6IFwiM1wiLFxuICAgICAgICAgICAgXCJjcmVhdGVUaW1lXCI6IFwiMjAyMy0xMC0yN1QxNjowMTowMFwiLFxuICAgICAgICAgICAgXCJjcmVhdGVVc2VyXCI6IFwiYWRtaW5cIixcbiAgICAgICAgICAgIFwiaWRcIjogMyxcbiAgICAgICAgICAgIFwiaXNEZWxcIjogMCxcbiAgICAgICAgICAgIFwidXNlcm5hbWVcIjogXCJ1c2VyXCIsXG4gICAgICAgICAgICBcInN0YXR1c1wiOiAxXG4gICAgICAgIH0sXG4gICAgICAgIHtcbiAgICAgICAgICAgIFwicm9sZUNuTmFtZVwiOiBcIuOAiuacquWIhumFjeOAi1wiLFxuICAgICAgICAgICAgXCJwYXNzd29yZFwiOiBcIlwiLFxuICAgICAgICAgICAgXCJjb2RlXCI6IFwidXNlcjEwMDAwMDAwNFwiLFxuICAgICAgICAgICAgXCJjcmVhdGVUaW1lXCI6IFwiMjAyMy0xMC0yN1QxODowOTozNlwiLFxuICAgICAgICAgICAgXCJ0ZWxcIjogMTIzMTIzLFxuICAgICAgICAgICAgXCJjcmVhdGVVc2VyXCI6IFwiXCIsXG4gICAgICAgICAgICBcImlkXCI6IDQsXG4gICAgICAgICAgICBcImlzRGVsXCI6IDAsXG4gICAgICAgICAgICBcImVtYWlsXCI6IFwiMTIzZGRAcXFxXCIsXG4gICAgICAgICAgICBcInVzZXJuYW1lXCI6IFwidGVzdDFcIixcbiAgICAgICAgICAgIFwic3RhdHVzXCI6IDFcbiAgICAgICAgfSxcbiAgICAgICAge1xuICAgICAgICAgICAgXCJjb2RlXCI6IFwidXNlcjEwMDAwMDAwOFwiLFxuICAgICAgICAgICAgXCJ1cGRhdGVVc2VyXCI6IFwiYWRtaW5cIixcbiAgICAgICAgICAgIFwidXBkYXRlVGltZVwiOiBcIjIwMjQtMDQtMTZUMTU6Mzg6NDhcIixcbiAgICAgICAgICAgIFwicm9sZUNuTmFtZVwiOiBcIuOAiuacquWIhumFjeOAi1wiLFxuICAgICAgICAgICAgXCJwYXNzd29yZFwiOiBcIlwiLFxuICAgICAgICAgICAgXCJjbk5hbWVcIjogXCLlubtcIixcbiAgICAgICAgICAgIFwiY3JlYXRlVGltZVwiOiBcIjIwMjQtMDQtMTVUMTU6MzI6NDNcIixcbiAgICAgICAgICAgIFwidGVsXCI6IDEyMzEzLFxuICAgICAgICAgICAgXCJjcmVhdGVVc2VyXCI6IFwiYWRtaW5cIixcbiAgICAgICAgICAgIFwiaWRcIjogNSxcbiAgICAgICAgICAgIFwiaXNEZWxcIjogMCxcbiAgICAgICAgICAgIFwiZW1haWxcIjogXCIxMjNAcXFcIixcbiAgICAgICAgICAgIFwidXNlcm5hbWVcIjogXCJodWFuXCIsXG4gICAgICAgICAgICBcInN0YXR1c1wiOiAxXG4gICAgICAgIH1cbiAgICBdXG48L3NjcmlwdD5cbiIsImltcG9ydC1tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHt9XG59IiwidHNjb25maWcuanNvbiI6IntcbiAgXCJjb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IFwiRVNOZXh0XCIsXG4gICAgXCJqc3hcIjogXCJwcmVzZXJ2ZVwiLFxuICAgIFwibW9kdWxlXCI6IFwiRVNOZXh0XCIsXG4gICAgXCJtb2R1bGVSZXNvbHV0aW9uXCI6IFwiQnVuZGxlclwiLFxuICAgIFwidHlwZXNcIjogW1wiZWxlbWVudC1wbHVzL2dsb2JhbC5kLnRzXCJdLFxuICAgIFwiYWxsb3dJbXBvcnRpbmdUc0V4dGVuc2lvbnNcIjogdHJ1ZSxcbiAgICBcImFsbG93SnNcIjogdHJ1ZSxcbiAgICBcImNoZWNrSnNcIjogdHJ1ZVxuICB9LFxuICBcInZ1ZUNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogMy4zXG4gIH1cbn1cbiIsIl9vIjp7fX0=

**Comment by warmthsea** [comment]
检查一下是否是版本问题或者是代码引起的问题

**Reply by Poom6zZ** [reply]
确实是版本问题，我的elementplus的版本是2.4.1，升级2.6.0之后的版本问题解决
